racket/src/LICENSE.txt

```diff
@@ -1,52 +1,86 @@
-Racket is distributed under the GNU Lesser General Public License
-(LGPL) version 3.  This implies that you may link Racket into
-proprietary applications, provided you follow the rules stated in the
-LGPL.  You can also modify Racket; if you distribute a modified
-version, you must distribute it under the terms of the LGPL, which in
-particular states that you must release the source code for the
-modified software.
+Racket is distributed under the MIT license and the Apache version 2.0
+license, at your option. However, Racket relies on some components
+distributed under other licenses. In short:
 
-See the file LICENSE-LGPL.txt for the full text of the license.
+  * The traditional Racket executable includes code distributed under
+  the GNU Lesser General Public License, version 3. This is the
+  executable built from code in racket/src/racket.
```

Comment:
"racket/src/racket" relative to this readme is just "racket". Several other relative paths below should be (re-)adjusted.